test/fixtures/script_number.json

```diff
@@ -0,0 +1,225 @@
+[
```

Comment:
These fixtures are [generated](https://gist.github.com/dcousens/5352f9488577631afd96) from bitcoin core.
@jprichardson any ideas on how we could increase transparency over this?

A link to a gist that will likely disappear soon isn't great.